sale_order_invoicing_grouping_criteria/views/sale_invoicing_grouping_criteria_views.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+    Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+    License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+-->
```

Comment:
This copyright?

Reply:
Copy/paste from the original Eficent module... I will amend